src/python/pants/backend/docker/util_rules/docker_build_context.py

```diff
@@ -50,6 +51,14 @@
 logger = logging.getLogger(__name__)
 
 
+class DockerfileImageTagsDeprecation(DeprecatedDockerInterpolationValue):
+    _removal_version = "2.11.0.dev0"
+    _hint = (
+        "The `<stage>.tag` version values are deprecated in favour of `tags.<stage>`.\n\n"
```

Comment:
Would be good to Save A Click and give an example:

> For example, rather than using `{baseimage.tag}` in your Docker template, use `{tags.baseimage}`.

 But then still link to this PR like you do if people still aren't understanding.